site/content/docs/5.1/utilities/opacity.md

```diff
@@ -3,6 +3,7 @@ layout: docs
 title: Opacity
 description: Control the opacity of elements.
 group: utilities
+added: "5.1"
```

Comment:
This needs to be added as well for "Stacks" (`site/content/docs/5.1/helpers/stacks.md`) and "Vertical rule" (`site/content/docs/5.1/helpers/vertical-rule.md`)